staker/challenge_test.go

```diff
@@ -298,3 +298,17 @@ func TestChallengeToFailedTooFar(t *testing.T) {
 	Require(t, machine.AddSequencerInboxMessage(10, []byte{0, 1, 2, 3}))
 	runChallengeTest(t, machine, incorrectMachine, true, false, 11)
 }
+
+// func TestReadHotShotOp(t *testing.T) {
```

Comment:
Delete?

Reply:
Add a comment now. This test is to test if the OpCode::ReadHotShotCommitment is working normally. But a bug found in Arbitrum and after they fix it, we should comment out this test.